Locker/Internal/Model/DeviceResponse.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DeviceResponse.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 20.10.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import Foundation
+
+struct DeviceResponse: Codable {
+    let devices: [String]
```

Comment:
I can see that in the project, you have this:
```swift
    let faceIdDevices: [String]
    let touchIdDevices: [String]
```

I'm not sure why this is not visible on GitHub so I'm leaving the comment just in case something is missing in the final codebase. Implementation from the project is correct one :)